README.md

```diff
@@ -1,63 +1,40 @@
+* Install Office 2016 for Windows and join the [Office Insider](https://products.office.com/en-us/office-insider) program. You must have Office build number 8711 or later.
 
 ## To use the project
 
-Follow these instructions to use this custom function sample add-in:
+On a machine with a valid instance of an Excel Insider build installed, follow these instructions to use this custom function sample add-in:
 
-1. Publish the code files (HTML, JS, and JSON) to localhost.
-2. Replace `http://127.0.0.1:8080` in the manifest file (there are 4 occurrences) with the URL you used, if needed (you might be using a different port number). 
-3. Sideload the manifest using the instructions found at <https://aka.ms/sideload-addins>.
-4. Test a custom function by entering `=CONTOSO.ADD42` in a cell.
-5. Try the other functions in the sample: `=CONTOSO.ADD42ASYNC`, `CONTOSO.ISPRIME`, `CONTOSO.NTHPRIME`, `CONTOSO.GETDAY`, `CONTOSO.INCREMENTVALUE`, and `CONTOSO.SECONDHIGHEST`.
-7. Follow @OfficeDev on Twitter for updates, post suggestions to the [Excel Add-ins UserVoice page](https://officespdev.uservoice.com/forums/224641-feature-requests-and-feedback/category/163563-add-in-excel) and tag Stack Overflow questions with the [custom-functions-excel](https://stackoverflow.com/questions/tagged/custom-functions-excel) tag.
+1. On the machine where your custom functions project is installed, follow the instructions to install the self-signed certificates (https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md) . 
+2. From a command prompt from within your custom functions project directory, run `npm run start` to start a localhost server instance. 
+3. Run `npm run sideload` to launch Excel and load the custom functions add-in. Additonal information on sideloading can be found at <https://aka.ms/sideload-addins>.
+4. After Excel launches, you will need to register the custom-functions add-in to work around a bug:
+    a. On the upper-left-hand side of Excel, there is a small hexagon icon with a dropdown arrow. The icon is to right of the Save icon.
+    b. Click on this dropdown arrow and then click on the Custom Functions Sample add-in to register it.
+5. Test a custom function by entering `=CONTOSO.ADD42(num)` in a cell.
+6. Try the other functions in the sample: `=CONTOSO.ADD42ASYNC(num, num)`, `CONTOSO.ISPRIME(num)`, `CONTOSO.NTHPRIME(num)`, `CONTOSO.GETDAY()`, `CONTOSO.INCREMENTVALUE(increment)`, and `CONTOSO.SECONDHIGHEST(range)`.
+7. If you make changes to the sample add-in, copy the updated files to your website, and then close and reopen Excel. If your functions are not available in Excel, re-insert the add-in using **Insert** > **My Add-ins**.
+8. Follow @OfficeDev on Twitter for updates and send feedback to <excelcustomfunctions@microsoft.com>.
 
-## Making changes
```

Comment:
Do we not need 36-60 any more? Just curious! They seem useful to me but maybe I'm lacking the context for why you removed them.